rsconnect/main.py

```diff
@@ -1247,6 +1334,94 @@ def write_manifest_notebook(
             write_environment_file(environment, base_dir)
 
 
+@write_manifest.command(
+    name="voila",
+    short_help="Create a manifest.json file for a Voila notebook.",
+    help=(
+        "Create a manifest.json file for a Voila notebook for later deployment. "
+        'This will create an environment file ("requirements.txt") if one does '
+        "not exist. All files are created in the same directory as the notebook file."
+    ),
+)
+@click.option("--overwrite", "-o", is_flag=True, help="Overwrite manifest.json, if it exists.")
+@click.option(
+    "--python",
+    "-p",
+    type=click.Path(exists=True),
+    help="Path to Python interpreter whose environment should be used. "
+    + "The Python environment must have the rsconnect package installed.",
+)
+@click.option(
+    "--conda",
+    "-C",
+    is_flag=True,
+    hidden=True,
+    help="Use Conda to deploy (requires RStudio Connect version 1.8.2 or later)",
+)
+@click.option(
+    "--force-generate",
+    "-g",
+    is_flag=True,
+    help='Force generating "requirements.txt", even if it already exists.',
+)
+@click.option("--verbose", "-v", "verbose", is_flag=True, help="Print detailed messages")
+@click.option(
+    "--image",
+    "-I",
+    help="Target image to be used during content execution (only applicable if the RStudio Connect "
+    "server is configured to use off-host execution)",
+)
+@click.argument("file", type=click.Path(exists=True, dir_okay=False, file_okay=True))
```

Comment:
Are we going to extend this to directory deployment? Maybe in a separate PR would be better.